websurfx/config.lua

```diff
@@ -19,3 +19,8 @@ theme = "simple" -- the theme name which should be used for the website
 
 -- Caching
 redis_connection_url = "redis://127.0.0.1:8082" -- redis connection url address on which the client should connect on.
+
+-- Aggregator
+aggregator = {
+  random_delay = false, -- whether to add random delay before sending the request to the search engine
```

Comment:
```suggestion
production_use = false
```

I feel that having an `aggregator` option with suboptions like `random_delay` will make it more ambiguous people will have questions like what is this `aggregator` option?? and Why is there a random delay ??. I think having a  `production_use` option will make it more self-documenting and less ambiguous for general users seeing the option itself they will get the idea like yes this option should only be set to true if it is to be used for large-scale use, also maybe in future we can think of adding more options under it but right now having `production_use` seems ok for now :slightly_smiling_face: .

Reply:
That make sense!